designer/client/components/Fee/FeeEdit.jsx

```diff
@@ -29,6 +49,9 @@ export class FeeEdit extends React.Component {
     const descriptions = formData.getAll("description").map((t) => t.trim());
     const amount = formData.getAll("amount").map((t) => t.trim());
     const conditions = formData.getAll("condition").map((t) => t.trim());
+    const multipliers = formData
+      .getAll("multiplier")
+      .map((t) => t.trim() ?? "0");
```

Comment:
set all the multipliers to 0?

I think this may have unintended consequences for anyone who has a working form and wants to update it. the multiplier should be set to 1 by default if we go ahead with this. it seems like when you add something, it's set to 1 anyway?